llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -9060,12 +8878,14 @@ collectUsersInExitBlocks(Loop *OrigLoop, VPRecipeBuilder &Builder,
 // exit users can be handled, otherwise return false.
 static bool
 addUsersInExitBlocks(VPlan &Plan,
-                     const SetVector<VPIRInstruction *> &ExitUsersToFix) {
+                     const SetVector<VPIRInstruction *> &ExitUsersToFix,
+                     DenseMap<VPValue *, VPValue *> &IVEndValues) {
```

Comment:
The `IVEndValues` parameter seems to be unused here.

Reply:
Should be gone now, thanks